strings.txt

```diff
@@ -16217,48 +16217,644 @@
 
   [editor_storey_number]
     tags = ios, android
-    en = Storey number (maximum %d)
+    en = Number of floors (max %d)
```

Comment:
Алекс, уверен? Перевод пришёл именно такой от переводчиков.

> On Apr 26, 2016, at 3:41 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61077704>:
> 
> > @@ -16217,48 +16217,644 @@
> >  
> >    [editor_storey_number]
> >      tags = ios, android
> > -    en = Storey number (maximum %d)
> > +    en = Number of floors (max %d)
> Building Levels (max %d)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295#r61077704>